packages/zen-bull-netting/test/unit-test/ZenBullNettingUnit.t.sol

```diff
@@ -139,4 +146,39 @@ contract ZenBullNettingUnit is ZenBullNettingBaseSetup {
         vm.expectRevert(bytes("ZBN02"));
         zenBullNetting.setOTCPriceTolerance(maxOtcPriceTolerance + 1);
     }
+
+    function testQueueWeth() public {
+        uint256 amount = 100e18;
+        uint256 user1WethBalanceBefore = IERC20(WETH).balanceOf(user1);
+        uint256 zenBullNettingWethBalanceBefore = IERC20(WETH).balanceOf(address(zenBullNetting));
+
+        vm.startPrank(user1);
+        IERC20(WETH).approve(address(zenBullNetting), amount);
+        zenBullNetting.queueWeth(amount);
+        vm.stopPrank();
+
+        assertEq(zenBullNetting.wethBalance(user1), amount);
+        assertEq(
+            IERC20(WETH).balanceOf(address(zenBullNetting)) - zenBullNettingWethBalanceBefore,
+            amount
+        );
+        assertEq(IERC20(WETH).balanceOf(user1) + amount, user1WethBalanceBefore);
+        assertEq(zenBullNetting.depositsQueued(), amount);
+    }
```

Comment:
should also check if the receipt is done correctly? 

ie msg.sender, timestamp recorded etc?

Reply:
Yeah didn't add those specific asserts as the code is the same in Crab netting, can add them later